.github/workflows/ci.yml

```diff
@@ -52,6 +52,7 @@ jobs:
             -DFAISS_DIR=$AMS_FAISS_PATH \
             -DWITH_AMS_DEBUG=On \
             -DWITH_WORKFLOW=On \
+            -DWITH_ADIAK=On \
```

Comment:
Can you please add a test case in which ADIAK is off?

Reply:
Done, updated one config in each job for adiak to be off